tools/projmgr/templates/cdefault.yml

```diff
@@ -30,7 +28,6 @@ default:
         - -std=gnu11
       Link:
         - --specs=nano.specs
-        - --specs=rdimon.specs
```

Comment:
This may break existing projects using semihosted stdout for printing analysis or test results.

Reply:
Understood, this will cause some friction in project using the cdefault from etc.